webpack.base.config.js

```diff
@@ -214,6 +215,7 @@ module.exports = (env) => {
               loader: 'sass-loader',
               options: {
                 implementation: require("sass"),
+                additionalData: `$is-codecombat: ${product == 'codecombat'};`,
```

Comment:
Don't we need `sassOptions` here as above?

Reply:
i don't know. but if it works normal, i don't want to change it. since i don't feel so familiar with webpack configs 😂

Reply:
I think we can use it this way: https://webpack.js.org/loaders/sass-loader/#string-1

Also, you can always add a class in HTML based on a variable in Vue template if you don't want to go via this route 😅

Reply:
oh sorry i don't catch you two yesterday. now it is fixed 😂